pytest_selenium/pytest_selenium.py

```diff
@@ -112,7 +112,6 @@ def capabilities(
             key = firefox_options.KEY
             options = firefox_options.to_capabilities()
         elif browser == "EDGE":
-            key = edge_options.KEY
```

Comment:
@nfk This was written with selenium 4.0 in mind: https://github.com/SeleniumHQ/selenium/blob/trunk/py/selenium/webdriver/edge/options.py

Does this brake something for you with the current selenium version?

If not we should revert this change. Else we should add a workaround checking the version or adding a try except.

Reply:
As notice in commit https://github.com/pytest-dev/pytest-selenium/pull/260/commits/1443b9c0029dd06755fa9d9aa27a03c47a3a2c8d. It does not work on selenium 3.141. I am going to do the change to support selenium 3.X and 4.X. Thanks for the context.

Reply:
@PhilippSelenium fixed, then I fixed the CI too.

Reply:
(y)